doc-links.yml

```diff
@@ -145,6 +145,8 @@
         link: /reference/ambassador-with-aws
       - title: Diagnostics
         link: /reference/diagnostics
+      - title: Debugging
+        link: /reference/debugging
```

Comment:
This is gonna conflict with the already merged https://github.com/datawire/ambassador-docs/pull/178